pages/ae/am/am-installation.adoc

```diff
@@ -86,3 +86,54 @@ alerts:
 
 CAUTION: By default, to keep the same behavior of the previous version, events are sent over a websocket connection.
 The default behavior will switch to http in a next future version.
+
+=== Connect to multiple engines
+
+Let's say you have to connect to multiple Alert Engines in order to propagate the triggers across multiple clusters.
+The connector lets you do this:
+
+```yaml
+alerts:
+  alert-engine:
+    enabled: true
+    engines:
+      cluster1:
+        endpoints:
+          - https://localhost:8072/
+        security:
+          username: admin
+          password: adminadmin
+        #ssl:
+        #  keystore:
+        #    type: jks # Supports jks, pem, pkcs12
+        #    path: /path/to/keystore.jks
+        #    password: changeit
+        #  truststore:
+        #    type: jks # Supports jks, pem, pkcs12
+        #    path: /path/to/truststore.jks
+        #    password: changeit
+      default:
+        endpoints:
+          - http://localhost:8073/
+        security:
+          username: admin
+          password: adminadmin
+        #ssl:
+        #  keystore:
+        #    type: pem # Supports jks, pem, pkcs12
+        #    certs:
+        #      - /path/to/cert.pem
+        #      - /path/to/cert2.pem
+        #    keys:
+        #      - /path/to/key.pem
+        #      - /path/to/key2.pem
+        #  truststore:
+        #    type: pem # Supports jks, pem, pkcs12
+        #    path: /path/to/truststore.pem
+```
```

Comment:
Maybe we can mention the fact that we don't need redundancy configuration on `ws` section.
Except to overwrite the `sendEventsOnHttp` attribute ?